src/components/Card.js

```diff
@@ -36,18 +38,25 @@ const intervalDate = (value) => {
   }
 };
 
-const Card = ({ cardlist }) => {
-  const { url, title, imageSource, description, createdAt } = cardlist;
+const Card = ({ card, isIconVisible = true }) => {
+  const { url, title, description } = card;
+  const createdAt = card?.createdAt ? card?.createdAt : card?.created_at;
+  const imageSource = card?.imageSource ? card?.imageSource : card?.image_source;
   const cardImage = imageSource ? imageSource : noneData;
-  const altCardImage = title + "로 이동"
-  console.log(altCardImage);
-
 
   return (
     <div className="card">
+      {isIconVisible ? (
+        <>
+          <img className="star-icon" src={iconStar} />
+          <img className="kebab-icon" src={iconKebab} />
+        </>
+      ) : (
+        null
+      )}
```

Comment:
이거는 조건부 렌더링으로 코드를 조금 줄일 수 있어요.
```suggestion
      {isIconVisible  &&  (
        <>
          <img className="star-icon" src={iconStar} />
          <img className="kebab-icon" src={iconKebab} />
        </>
      ) }
```

Reply:
a0472f09c050f359437447f3843aea650c2ee037 정정했습니도~!